src/spring-cloud/azext_spring_cloud/custom.py

```diff
@@ -99,6 +99,7 @@ def app_create(cmd, client, resource_group, service, name,
                instance_count=None,
                runtime_version=None,
                jvm_options=None,
+               net_core_main_entry_path=None,
```

Comment:
We don't need to set NetCoreMainEntryPath during app creation, this should always be `null`.